web/components/requests/FcDetailsStudyRequest.vue

```diff
@@ -408,6 +445,38 @@ export default {
         this.internalValue.dueDate = this.dueDate;
       }
     },
+    location() {
+      this.updateStudyRequestLocation();
```

Comment:
can we get rid of this?  i don't think we want to allow location editing in Request Study - which also means that we'll need to lock / hide certain features on `FcPaneMap` (but we can do that part in #599 ).